pkgs/applications/misc/octoprint/default.nix

```diff
@@ -113,13 +110,13 @@ let
           self: super: {
             octoprint = self.buildPythonPackage rec {
               pname = "OctoPrint";
-              version = "1.5.3";
+              version = "1.6.1";
 
               src = fetchFromGitHub {
                 owner = "OctoPrint";
                 repo = "OctoPrint";
                 rev = version;
-                sha256 = "sha256-ZL/P/YIHynPmP8ssZZUKZDJscBsSsCq3UtOHrTVLpec=";
+                sha256 = "0mrh7z38wyjfliq2icl4pzb196n5k7r0yxd91h7jvyajc0zmfrvw";
```

Comment:
```suggestion
                sha256 = "04rwm2s6x2c4b320ylqvx0cd5n8089xabzzxzigjmx873zvf9gfx";
```

Reply:
Not quite sure what happened here, but I updated the hash.